DeepSpeech.py

```diff
@@ -3,179 +3,168 @@
 from __future__ import absolute_import
 # -*- coding: utf-8 -*-
 
+import os
+
+if 'TF_CPP_MIN_LOG_LEVEL' not in os.environ:
+    os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
 import datetime
 import json
 import numpy as np
-import os
 import shutil
 import subprocess
 import sys
 import tempfile
 import tensorflow as tf
 import time
+import importlib
+import BaseHTTPServer
+import urllib
+import urllib2
```

Comment:
These two lines need to be combined into `from six.moves import urllib`, preferably as part of the existing six.moves import below, for Python 2/3 compat.